formulaic/materializers/transforms/basis_splines.py

```diff
@@ -0,0 +1,86 @@
+"""
+Modified version of code originally from Zepid, Copyright (c) 2018 Paul Zivich.
```

Comment:
Note this. I've worked on Zepid with Paul, and I've modified this splines code there too. About 80% of the code below is from that project - is this reference enough?

Reply:
I always feel a bit worried when code is copied and pasted from external code, especially when not by the original author... **but** the MIT license explicitly allows it, and you have worked with the original author, so I don't think that will be a problem here :).